config/http_config.go

```diff
@@ -21,6 +21,7 @@ import (
 	"crypto/x509"
 	"encoding/json"
 	"fmt"
+	"github.com/icholy/digest"
```

Comment:
Move this to line 33 and keep a blank line before.
golangci-lint run --fix shall do the job